compiler/Notes

```diff
@@ -0,0 +1,6 @@
+How I installed Scala:
+
```

Comment:
These notes don't belong here. Should we add the instructions to `compiler/README.adoc`?

Reply:
I have removed Notes